executing/executing.py

```diff
@@ -235,8 +258,8 @@ def __init__(self, filename, lines):
         else:
             for node in ast.walk(self.tree):
                 for child in ast.iter_child_nodes(node):
-                    child.parent = node
-                for lineno in node_linenos(node):
+                    cast(EnhancedAST, child).parent = cast(EnhancedAST, node)
+                for lineno in node_linenos(cast(EnhancedAST, node)):
```

Comment:
Both calls to node_linenos have this cast. But node_linenos accepts AST and doesn't need the parent.

Reply:
Removed the cast